CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
-## 5.6.1-SNAPSHOT (future release)
+## 5.7.0-SNAPSHOT (future release)
```

Comment:
Also needs a corresponding version bump to pom.xml files.

Reply:
Done